content/venue.md

```diff
@@ -9,10 +9,56 @@ The main conference venue is Cardiff’s beautiful [City
 Hall](http://www.cardiffcityhall.com/), with evening events held around the
 centre of Cardiff.
 
-### Getting to the venue
+See [Accommodation](/accommodation) for recommended hotels and other places to stay.
 
-There is information on the [venue website](http://www.cardiffcityhall.com/find-us)
-about getting to Cardiff, getting to the venue, and parking.
+## Cardiff City Hall
 
-For those travelling from overseas, there is a small airport at Cardiff, and a
-larger airport at Bristol.
+PyCon UK 2016 will take place at [Cardiff City Hall](http://www.cardiffcityhall.com/find-us), Cathays Park, Cardiff
+CF10 3ND. Cardiff City Halls is accessible for people of limited mobility. If you need to park near the venue because
+of mobility issues, please contact us and we'll do our best to help.
+
+## Getting to Cardiff
+
+### Trains
+
+Trains to Cardiff arrive right in the city centre, at Cardiff Central railway station. Unfortunately, engineering work
+at the Severn Tunnel means that trains around the time of the conference will be disrupted, and some parts of affected
+journeys may be replaced by bus services.
```

Comment:
Indeed. They chose our precise weekend - and announced it after we announced ours!